app/javascript/components/admin/site_settings/settings/Settings.jsx

```diff
@@ -21,12 +21,14 @@ import useSiteSettings from '../../../../hooks/queries/admin/site_settings/useSi
 import SettingsRow from '../SettingsRow';
 import SettingSelect from './SettingSelect';
 import useUpdateSiteSetting from '../../../../hooks/mutations/admin/site_settings/useUpdateSiteSetting';
+import useEnv from '../../../../hooks/queries/env/useEnv';
 
 export default function Settings() {
   const { t } = useTranslation();
   const { data: siteSettings, isLoading } = useSiteSettings(['ShareRooms', 'PreuploadPresentation', 'DefaultRecordingVisibility', 'SessionTimeout']);
   const updateDefaultRecordingVisibility = useUpdateSiteSetting('DefaultRecordingVisibility');
   const updateSessionTimeout = useUpdateSiteSetting('SessionTimeout');
+  const envAPI = useEnv();
```

Comment:
You can break this down for simplicity `const { data: env } = useEnv();` so that below you can just do `env?.EXTERNAL_AUTH` (without the data)